posts/dask-summit-takeaway.md

```diff
@@ -53,6 +70,40 @@ This session was a keynote focused on how these packages + ecosystem grew
 - **_Provide scope and build people_**
   - Move towards "grassroots" not so much top down
 
+## Xarray User Forum + Updates
+
+- Xarray now includes a backend api, allowing contributors to add new backends for reading + operating on datasets
+  - Including the `raw_indexing` method is important to enable lazy loading
+- Flexible indexing is coming to xarray soon
+  - Helps when dealing with irregular spaced data, working with projected coordinate systems, staggered grids
+  - Currently, Xarray uses pandas for indexing, which is problematic when wanting to do flexible indexing
+  - Current Xarray accessor - [Xoak](https://xoak.readthedocs.io/en/latest/)
+    - See [previous blog post on working with unstructured grids](https://ncar.github.io/esds/posts/multiple_index_xarray_xoak/)
+  - See the [design documents for flexible indexing in Xarray](https://github.com/pydata/xarray/blob/master/design_notes/flexible_indexes_notes.md)
+- Duck arrays are now supported in Xarray
+  - Helps when working with [pint](https://pint.readthedocs.io/en/stable/)
+  - **_Most_** of the xarray API supports duck arrays, few aspects that still don't
+
+### [Xarray User Forum Talk Links](https://summit.dask.org/schedule/presentation/51/xarray-user-forum/)
+
+- Aureliana Barghini’s Talk
+
+  - [New backend plugin infrastructure](https://github.com/aurghs/xarray-backend-tutorial)
+
+- Links from Benoit Bovy’s talk
+
+  - [Flexible indexes design notes](https://github.com/pydata/xarray/blob/master/design_notes/flexible_indexes_notes.md)
+  - [Explicit indexes project](https://github.com/pydata/xarray/projects/1)
+
+- Davis Bennett’s talk
+
+  - [Hierarchical storage data structure in xarray](https://github.com/pydata/xarray/issues/4118)
+
+- Ravi Kumar’s talk:
+  - [Using Xarray to store statistical diagnostic data](https://github.com/arviz-devs/arviz_misc/tree/master/xarray_user_forum_2021)
+- Jack’s talk:
+  - [Xarray Fusion Research](https://xyzpy.readthedocs.io/en/latest/)
```

Comment:
As far as I can remember, `xyzpy` isn't related to Jack's talk. It was just mentioned in a comment by Stephan Hoyer in regards to how this package has some nice functionality for constructing complex xarray datasets...